pkgs/by-name/ta/taschenrechner/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, rustPlatform
+, fetchFromGitLab
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "taschenrechner";
+  version = "1.3.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.fem-net.de";
+    owner = "mabl";
+    repo = "taschenrechner";
+    rev = version;
+    hash = "sha256-PF9VCdlgA4c4Qw8Ih3JT29/r2e7i162lVAbW1QSOlWo=";
+  };
+
+  cargoHash = "sha256-SFgStvpcqEwus1JBs5ZyMHO1UD0oWV7mvS6o4v5gIFc=";
+
+  meta = with lib; {
+    description = "A cli-calculator written in Rust";
+    homepage = "https://gitlab.fem-net.de/mabl/taschenrechner";
+    license = licenses.gpl3;
```

Comment:
`gpl3` is [deprecated for being too unclear](https://github.com/NixOS/nixpkgs/commit/9d5bd8b835ddd62db5857b86fcda5af00c237bfa), can you clarify if `gpl3Only` or `gpl3Plus`?

Reply:
oh yeah you are right.

Reply:
The cargo.toml says gpl3.0 https://gitlab.fem-net.de/mabl/taschenrechner/-/blob/main/Cargo.toml#L5 so I guess gpl3Only

Reply:
Upstream sadly has neither a copyright note in their code files nor a copy of the license to clarify their exact wishes… `gpl3Only` would be my guess as well, just to be safe.

Reply:
Fixed